pkg/ctl/scale/nodegroup.go

```diff
@@ -44,6 +45,10 @@ func doScaleNodeGroup(rc *cmdutils.ResourceCmd, ng *api.NodeGroup) error {
 
 	ctl := eks.New(rc.ProviderConfig, cfg)
```

Comment:
This command doesn't support `--config-file`, so we don't do defaulting/validation here.

Reply:
maybe add that as a comment?

Reply:
maybe add this as a comment?